server/controllers/configurationsController/configurationsController.test.js

```diff
@@ -0,0 +1,184 @@
+import ConfigurationsController from '.'
+import { createRequestWithUser, createResponse } from '../../../test/fixtures'
+
+describe('ConfigurationsController', () => {
+  describe(ConfigurationsController.create, () => {
+    const newConfiguration = { body: { name: 'new matrix', owner: 'owl' } }
+
+    describe('When successful', () => {
+      const request = createRequestWithUser(newConfiguration)
+      const response = createResponse()
+
+      it('sends a status of 200 with a data property when successful', async () => {
+        request.app.locals.appDb.insertOne.mockResolvedValueOnce({
+          insertedId: 'eagle',
+        })
+
+        await ConfigurationsController.create(request, response)
+
+        expect(response.status).toHaveBeenCalledWith(200)
+        expect(response.json).toHaveBeenCalledWith({
+          data: 'eagle',
+        })
```

Comment:
this should return the created configuration record

Reply:
[f11e811](https://github.com/AMP-SCZ/dpdash/pull/305/commits/f11e811a37bb7df8cc158a8d0e061615e8ec535d)